File: 2048Game/Player.cpp
#include <bits/stdc++.h>

using namespace std;

int main(int arg, char **args) {

    int n, m;

    cin >> n >> m;

    int a[n][m];

    for (int i = 0; i < n; i++)
        for (int j = 0; j < m; j++)
            cin >> a[i][j];

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++)
            cout << a[i][j] << " ";
	cout << endl;
    }

    srand (time(NULL));

    switch (rand()%4) {
        case 0:
            cout << "up\n";
            break;
        case 1:
            cout << "down\n";
            break;
        case 2:
            cout << "left\n";
            break;
        case 3:
            cout << "right\n";
            break;
        default:
            break;
    }

    return 0;
}

File: 2048Game/Player.java
import java.util.*;
import java.io.*;

public class Player {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        int[][] a = new int[n][m];

        for (int i = 0; i < n; i++)
            for (int j = 0; j < m; j++)
                a[i][j] = sc.nextInt();

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++)
                System.out.print(a[i][j] + " ");
            System.out.println();
        }

        Random rand = new Random();

        switch (rand.nextInt(4)) {
            case 0:
                System.out.println("up");
                break;
            case 1:
                System.out.println("down");
                break;
            case 2:
                System.out.println("left");
                break;
            case 3:
                System.out.println("right");
                break;
            default:
                break;
        }
    }
}